app/models/bookable_slot.rb

```diff
@@ -8,26 +8,26 @@ def self.regenerate_for_six_weeks
   end
 
   def self.within_date_range(from, to)
-    where('usable_slots.start_at > ? AND usable_slots.end_at < ?', from, to)
+    where('bookable_slots.start_at > ? AND bookable_slots.end_at < ?', from, to)
```

Comment:
In lots of cases where we're explicitly providing the table we don't actually need to, thus it makes changes like this more work than renaming the table / class.